### PR TITLE
DTGB-432: Added stubs for 403 and 404 pages + some template changes t…

### PR DESCRIPTION
**Pull request by meirege**
…o info pages.

## Description
Some template changes. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/61-layouts/general/info-layout/_detail-layout.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.detail-layout {
+  @extend %content-container;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 1.2rem;
+  font-size: .9rem;
```
Comment (delrueba) [inline styleguide/components/61-layouts/general/info-layout/_detail-layout.scss:8]: Isn't there's a mixin/class/thingy to handle text sizes?
I feel this should be handled on a higher level.

File: styleguide/components/61-layouts/general/info-layout/_detail-layout.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  > .content {
+    width: span(8 of 12);
+    max-width: 36rem;
+    padding-right: span(1 of 12);
+
+    .image-gallery,
+    .video,
+    .quote,
+    .feedback-form {
+      @extend %wider;
```
Comment (delrueba) [inline styleguide/components/61-layouts/general/info-layout/_detail-layout.scss]: Reverse the logic
You've already got four components in the exception and only one component as default.

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig [hunk 3]
```diff
@@ -1,13 +1,50 @@
-<main class="info-page-layout">
+{#
```
Comment (delrueba) [inline styleguide/components/71-pages/general/info-page/info-page--variant1.twig]: no commented code

File: styleguide/components/71-pages/system/system-403-page/system-403-page.twig [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{#
```
Comment (delrueba) [inline styleguide/components/71-pages/system/system-403-page/system-403-page.twig]: no commented code

File: styleguide/components/71-pages/system/system-404-page/system-404-page.twig [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{#
```
Comment (delrueba) [inline styleguide/components/71-pages/system/system-404-page/system-404-page.twig]: no commented code

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/61-layouts/general/info-layout/_detail-layout.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/61-layouts/general/info-layout/_detail-layout.scss:8]: I'll create a refactor ticket for this. It doesn't belong in this branch.

File: styleguide/components/71-pages/general/info-page/info-page--variant1.twig (hunk 3 above)
Comment (meirege) [reply styleguide/components/71-pages/general/info-page/info-page--variant1.twig]: Fixed.

File: styleguide/components/71-pages/system/system-403-page/system-403-page.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/71-pages/system/system-403-page/system-403-page.twig]: Fixed.

File: styleguide/components/71-pages/system/system-404-page/system-404-page.twig (hunk 5 above)
Comment (meirege) [reply styleguide/components/71-pages/system/system-404-page/system-404-page.twig]: Fixed.

File: styleguide/components/61-layouts/general/info-layout/_detail-layout.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/61-layouts/general/info-layout/_detail-layout.scss]: I changed the logic.

File: styleguide/components/61-layouts/general/info-layout/_detail-layout.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/61-layouts/general/info-layout/_detail-layout.scss:8]: I've created ticket DTGB-435 for this purpose.

**Review by delrueba — APPROVED** [review]

**Pull request merged**